tools/gm-to-obj/README.md

```diff
@@ -0,0 +1,47 @@
+# .GM to .OBJ file converter
+
+## Requirements
+
+In order to run this script you need:
+
+- Python 3
+- Some .gm files from a storm game (TEHO, COAS, ...)
+
+## How to
+
+### Convert 1 file
+
+To run the script, just type in the console
+``python .\gm_to_obj_converter.py {filepath}`` 
+
+E.g. ``python .\gm_to_obj_converter.py "E:\SteamLibrary\steamapps\common\Sea Dogs To Each His Own\RESOURCE\MODELS\fish01.gm" -o myCoolFish.obj``
+
+Option | Short | Description | Default
+------ | ----- | ----------- | -------
+``--output``|``-o``| The name of the ``.obj`` file to create| [fileName] with ``.obj`` extension
+``--script-folder``|``-sf``|Put the output file in the same folder as the script|False
+``--quiet``|``-q``|Stop printing status|False
+
+### Convert all files
+
+This script will convert all files in one folder and subfolders. It will also print a list of failed items in a .txt file
+
+Run ``python .\convert_all.py {folder}``
+
+E.g. ``python .\convert_all.py "E:\SteamLibrary\steamapps\common\Sea Dogs To Each His Own\RESOURCE\MODELS\Ammo"``
+
+## Issues
+
+The following files will not convert, as they cannot be representend as .obj:
+
+TEHO
+
+- `big_sky.gm`
```

Comment:
@mitrokosta any updates on whether these are used or not?
@MangioneAndrea if there are no other known issues with the script, I think we can merge it in its current state.

Reply:
There are those 2 problems metioned by @Hammie .The mirroring looks important, if it's not urgent to merge it, i'd like to dive into it first

Reply:
The x-axis was indeed inverted, at least for items in TEHO, now that problem is fixed.
About the other one, I cannot reproduce it, so I'd say that it can be merged now